runtime/parachains/src/paras.rs

```diff
@@ -1004,7 +1004,9 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// Called by the initializer to finalize the configuration pallet.
```

Comment:
Not your fault, but the comment is c&p?

Reply:
yeah, but it does look correct, or?

Reply:
Is this the configuration pallet?

Reply:
ah gotcha xD will fix

Reply:
https://github.com/paritytech/polkadot/pull/4646